package.json

```diff
@@ -26,6 +26,7 @@
   },
   "license": "MIT",
   "dependencies": {
-    "basic-auth": "^2.0.0"
+    "basic-auth": "^2.0.0",
+    "tsscmp": "^1.0.6"
   }
```

Comment:
yes because out of the box you need the length comparison (just as an example) to properly do timing safe string comparison _even with_ Node's built-in method `crypto.timingSafeEqual`